mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -6,23 +6,25 @@ import { getApis, getExperiment, getParams, getRunInfos, getRunTags } from '../r
 import 'react-virtualized/styles.css';
 import { withRouter } from 'react-router-dom';
 import Routes from '../Routes';
-import { Button, ButtonGroup, Dropdown, DropdownButton, MenuItem } from 'react-bootstrap';
+import { Button, ButtonGroup, DropdownButton, MenuItem } from 'react-bootstrap';
 import { Experiment, RunInfo } from '../sdk/MlflowMessages';
 import { SearchUtils } from '../utils/SearchUtils';
 import classNames from 'classnames';
 import { saveAs } from 'file-saver';
 import { getLatestMetrics } from '../reducers/MetricReducer';
 import KeyFilter from '../utils/KeyFilter';
-import Utils from '../utils/Utils';
 
-import ExperimentRunsTableMultiColumn from "./ExperimentRunsTableMultiColumn";
-import ExperimentRunsTableCompact from "./ExperimentRunsTableCompact";
-import ExperimentViewUtil from "./ExperimentViewUtil";
-import ExperimentRunsSortToggle from "./ExperimentRunsSortToggle";
+import ExperimentRunsTableMultiColumnView from "./ExperimentRunsTableMultiColumnView";
+import ExperimentRunsTableCompactView from "./ExperimentRunsTableCompactView";
```

Comment:
Small bug: I think we need to rename `ExperimentRunsTableCompact.js` -> `ExperimentRunsTableCompactView.js`